ansible/playbooks/adhoc/grow_root_vg/grow_root_vg-m5.yml

```diff
@@ -179,6 +181,14 @@
   - name: Run partprobe to see if the size changed
     command: partprobe
 
+  - name: Get partition info (updated)
+    parted: 
+      device: "{{ vol_instance.fullname }}"
+      number: "{{ cli_var_partition_number }}"
+      state: info
+      unit: s
+    register: parted_info
```

Comment:
We talked a little bit about failing early if the playbook detects that new partitioning scheme is present to prevent clobbering those nodes. I don't think that necessarily needs to be present for this PR to be approved but wanted to see if this point in the playbook would be where to do that?

Reply:
It would be, yes.  You can check length of the partitions array.  I don't have the exact json handy.

For `/var/log` being split out, I did look at that and there is still only two partitions.  `/var/log` is just a logical volume.  So I don't think we need to spend time to add a check here.  If we need to use this playbook again there's a cli arg to indicate the partition you wish to grow, defaulting to 2.